packages/components/src/Banner/Banner.tsx

```diff
@@ -2,10 +2,10 @@ import Heading, { HeadingElement } from "../Heading";
 import Button from "../Button";
 import CheckCircleIcon from "../Icons/CheckCircle";
 import CloseIcon from "../Icons/Close";
+import DangerousRoundedIcon from "../Icons/custom/DangerousRounded";
```

Comment:
keeping these named `*Rounded` in case we change our minds and need a mix of multiple sets in the future or something 🤓